examples/hikari_lightbulb_basic_queue/requirements.txt

```
@@ -0,0 +1,2 @@
+hikari==2.0.0.dev102
+lightbulb==1.3.0
```

```suggestion
lightbulb==1.3.1
```

---

IDK which of the 2 will work.

```suggestion
hikari-lightbulb==1.3.1
```